bullet_train-sortable/app/javascript/controllers/sortable_controller.js

```diff
@@ -1,88 +1,337 @@
 import { Controller } from "@hotwired/stimulus"
```

Comment:
This is really a whole new controller, but it looks like a bunch of changes to `sortable_controller.js` because `git` is kinda dumb about replacing the entire contents of files. It may be easier to review by looking at the raw source of the controller rather than the diff.

Reply:
I think that's what I did. Or maybe I meant to and didn't...